docs/user/get-started/ios.md

```diff
@@ -0,0 +1,105 @@
+# Adding Glean to your iOS project
+
+## Usage
+
+### Integrating with your project
+
+#### Setting up the dependency
+
+Glean can be consumed through [Carthage](https://github.com/Carthage/Carthage), a dependency manager for macOS and iOS.
+For consuming the latest version of Glean, add the following line to your `Cartfile`::
+
+```
+github "mozilla/glean" "master"
+```
+
+#### Integrating with the build system
+
+In order for the Glean SDK to generate an API for your metrics, a Python environment must be accessible at build time.
+
+**TODO: Fill in how to add it to the build system**
```

Comment:
Do we have a bug for this? If so, can we reference it here?

Reply:
I plan to fill this in a bit before landing, but I'll also file a bug for expanding.